src/main/scala/fr/polytechnique/cmap/cnam/etl/events/HospitalStay.scala

```diff
@@ -27,4 +27,16 @@ trait HospitalStay extends AnyEvent with EventBuilder {
     apply(patientID, hospitalID, hospitalID, 0D, start, Some(end))
 }
 
-object HospitalStay extends HospitalStay
\ No newline at end of file
+object HospitalStay extends HospitalStay
+
+object McoHospitalStay extends HospitalStay {
+  override val category: EventCategory[HospitalStay] = "mco_hospital_stay"
+}
+
+object SsrHospitalStay extends HospitalStay {
```

Comment:
Do we really have the concept of Hospital Stay in SSR?

Reply:
I think so. Everybody we worked with was talking about Ssr (or Had) health cares as hospitalStays. Despite being different hospital types as classical MCO, these should be referred to as hospitalStays as well.

Reply:
It has to be documented IMO. The concept is bit unnatural when considering the SSR & HAD meanings. Could you put a link or a documentation in this purpose?

Reply:
I added a small commentary and a reference to the collaborative documentation